testing/trino-product-tests/src/main/java/io/trino/tests/product/TestConfiguredFeatures.java

```diff
@@ -48,6 +48,8 @@ public void selectConfiguredConnectors()
         }
         String sql = "SELECT DISTINCT connector_name FROM system.metadata.catalogs";
         List<String> loadedCatalogs = onTrino().executeQuery(sql).column(1).stream().map(Object::toString).collect(toImmutableList());
-        assertThat(configuredConnectors).containsExactlyInAnyOrder(loadedCatalogs.toArray(new String[0]));
+        // for now loki connector is not properly loaded, when this is fixed this test will fail
```

Comment:
Please create a ticket or maybe let's remove loki from loading then. Is it used at all?

Nice, that you uncovered an issue.